source/game.database.bmx

```diff
@@ -1209,46 +1211,104 @@ Type TDatabaseLoader
 
 
 		'=== LOCALIZATION DATA ===
-		programmeData.title.Append( GetLocalizedStringFromNode(xml.FindElementNode(node, "title")) )
-		'programmeData.originalTitle.Append( GetLocalizedStringFromNode(xml.FindElementNode(node, "originalTitle")) )
-		programmeData.description.Append( GetLocalizedStringFromNode(xml.FindElementNode(node, "description")) )
+		If reuseExistingData
+			If Not programmeLicence.title
+				programmeLicence.title = GetLocalizedStringFromNode(xml.FindElementNode(node, "title"))
+			Else
+				programmeLicence.title.Append( GetLocalizedStringFromNode(xml.FindElementNode(node, "title")) )
+			EndIf
+			If Not programmeLicence.description
+				programmeLicence.description = GetLocalizedStringFromNode(xml.FindElementNode(node, "description"))
+			Else
+				programmeLicence.description.Append(GetLocalizedStringFromNode(xml.FindElementNode(node, "description")) )
+			EndIf
+		Else
+			programmeData.title.Append( GetLocalizedStringFromNode(xml.FindElementNode(node, "title")) )
+			'programmeData.originalTitle.Append( GetLocalizedStringFromNode(xml.FindElementNode(node, "originalTitle")) )
+			programmeData.description.Append( GetLocalizedStringFromNode(xml.FindElementNode(node, "description")) )
+		EndIf
 
 
 		'=== DATA ===
 		Local nodeData:TxmlNode = xml.FindChild(node, "data")
 		Local data:TData = New TData
-		xml.LoadValuesToData(nodeData, data, [..
-			"country", "distribution", "blocks", ..
-			"maingenre", "subgenre", "price_mod", ..
-			"available", "flags", "licence_flags", ..
-			"broadcast_time_slot_start", "broadcast_time_slot_end", ..
-			"broadcast_limit", "licence_broadcast_limit", ..
-			"broadcast_flags", "licence_broadcast_flags" ..
-		]) 'also allow a "<live>" block
-		'], ["live"]) 'also allow a "<live>" block
+		If reuseExistingData
+			'blocks cannot be overridden (only in data)
```

Comment:
Hmm ... das ist natuerlich bloed. Gerade Director's Cuts sind ja oft laenger. Auch koennten "Kritiken" anders ausfallen etc.

Reply:
In dem Fall wäre es vielleicht besser, den Director's Cut direkt als neues Programm zu definieren. Titel und Beschreibung sind ja am Ende vielleicht auch modifiziert, die Zielgruppenattraktivität ebenso...